package.json

```diff
@@ -41,30 +42,31 @@
     "html-loader": "0.4.3",
     "html-webpack-plugin": "2.22.0",
     "http-proxy-middleware": "0.17.1",
+    "immutable": "^3.8.1",
     "jest": "15.1.1",
     "json-loader": "0.5.4",
     "object-assign": "4.1.0",
     "opn": "4.0.2",
     "path-exists": "3.0.0",
     "postcss-loader": "0.13.0",
     "promise": "7.1.1",
+    "react-router": "~3.0.0",
```

Comment:
@mareklibra do we currently use react-router in ovirt-web-ui? If not I would avoid this until we properly introduce it (with v4 of react-router)